Classes/Domain/Repository/EmployeeRepository.php

```diff
@@ -91,7 +91,7 @@ public function getQueryBuilderToFindAllEntries(int $office = 0): QueryBuilder
 
         // Do not set any SELECT, ORDER BY, GROUP BY statement. It will be set by glossary2 API
         $queryBuilder
-            ->from($table, 'c')
+            ->from($table, 'e')
```

Comment:
The alias is only needed if you use JOINs. Here is no JOIN, so you can remove all aliases.